jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BuildDockerTask.java

```diff
@@ -142,4 +138,30 @@ BuildDockerTask setJibExtension(JibExtension jibExtension) {
     this.jibExtension = jibExtension;
     return this;
   }
+
+  /**
+   * Returns an {@link ImageReference} retrieved from the build configuration, or the project name/
```

Comment:
Something like `or {@code <the project name>/<version>}` to make its format clear, or what you did in `@return`